LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2StrategyElement_DefaultCovertActions_LW.uc

```diff
@@ -68,6 +69,31 @@ static function X2DataTemplate CreateIntenseTrainingTemplate()
 	return Template;
 }
 
+static function X2DataTemplate CreateResistanceMecTemplate()
+{
+	local X2CovertActionTemplate Template;
+
+	`CREATE_X2TEMPLATE(class'X2CovertActionTemplate', Template, 'CovertAction_ResistanceMec');
+
+	Template.ChooseLocationFn = ChooseRandomRegion;
+	Template.OverworldMeshPath = "UI_3D.Overwold_Final.CovertAction";
+
+	Template.Narratives.AddItem('CovertActionNarrative_ResistanceMec_Skirmishers');
+	Template.Narratives.AddItem('CovertActionNarrative_ResistanceMec_Reapers');
+	Template.Narratives.AddItem('CovertActionNarrative_ResistanceMec_Templars');
+
+    // NOTE: Soldier slots configured in `X2LWCovertActionsModTemplate` for consistency and
+    // because some Firaxis developer keeps making functions private..... The template mod
+    // also adds the Failure risk.
+
+	Template.Risks.AddItem('CovertActionRisk_Ambush');
+	Template.Risks.AddItem('CovertActionRisk_SoldierCaptured');
```

Comment:
You want to have a capture risk for this? I don't know if the reward is worth it. I don't remember any other "easy" covert actions having a capture risk.

Reply:
I couldn't remember myself. I'm okay with removing that.